.github/workflows/k8s_e2e.yml

```diff
@@ -40,6 +40,7 @@ env:
   RUST_TEST_THREADS: 1
   TEST_LOG: vector=debug
   VERBOSE: true
+  DISABLE_MOLD: true
```

Comment:
We disable installing/configuring `mold` on the test runner host for the Kubernetes E2E tests _because_ `cross` carries over Cargo configuration to the build container, which means the `rustc` wrapper, and `mold`, would need to exist in the build container... and also support the target triple.

It's just easier to skip it in that case. Other usages of `cross` don't actually install/configure `mold` on the test runner host (i.e. they don't run `bootstrap-ubuntu-20.04.sh`) so we don't need to tweak those workflows to disable it.

Reply:
That said, we might not even need to run `bootstrap-ubuntu-20.04.sh` on the test runner host... maybe that's just a historical hold-over. I'm not sure, and I'm leaving it as a future potential optimization.